.github/workflows/ci.yml

```
@@ -0,0 +1,58 @@
+on: [push, pull_request]
+
+name: CI
+
+env: 
+  RUST_BACKTRACE: 1
+
+jobs:
+  check:
+    name: cargo check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v3
        with:
          lfs: 'true'
```
that will actually check out the lfs files :-)